constraints/miden-vm/memory.air

```diff
@@ -0,0 +1,78 @@
+mod MemoryAir
+
+### Helper functions/evaluators ###################################################################
+
+# Enforces that columns must be zero.
+ev equal_zero(&main: [4]):
```

Comment:
The syntax doesn't look correct here. I think we need named columns here and we shouldn't use `&`? Was there a discussion on using this syntax that maybe I missed?

nit: Should we rename this to `is_zero`?